Project_0.Altis/SideStat/askSurrenderedUnitToJoin.sqf

```diff
@@ -0,0 +1,32 @@
+#include "..\OOP_Light\OOP_Light.h"
+
+// Client side only
+if (!hasInterface) exitWith {};
+params ["_target", "_caller", "_actionId", "_arguments"];
+
+[[_target, _actionId], {
+	if (!hasInterface) exitWith {};
+	params ["_target", "_actionId"];
+	_target removeAction _actionId;
+}] remoteExec ["call", 0, false];
+
+
+// This is sent to server
+[[_caller, _target], {
+	params ["_caller", "_target"];
+	
+	private _maybe = random 2;
+	private _sentence = "";
+	if (_maybe > 1) then {
+		CALLM1(SideStatWest, "incrementHumanResourcesBy", 1);
+		gSideStatWestHR = CALLM0(SideStatWest, "getHumanResources");
+		publicVariable "gSideStatWestHR";
+
+		_sentence = "ok cunt ill fight for ya"; 
+	} else {
+		_sentence = "fuck ya mate"; 
+	};
+
+	[_target, _sentence, _caller, false] remoteExecCall ["Dialog_fnc_hud_createSentence", owner _caller];
+
+}] remoteExec ["spawn", 2, false];
```

Comment:
I thought that remoteExec already spawns code? Need to check that some day.
Not a problem, just a comment.

Reply:
From wiki:
> Scripted function - scheduled environment (suspension is allowed, i.e. spawn, execVM).
> Script command - unscheduled environment (suspension is NOT allowed). 

So in this case the spawn is redundant it appears.

Reply:
Yeah could probably just remoteExec like this `parameters remoteExec ["fnc_doStuff"`, but right now we are sending whole code piece as parameter.

Reply:
ye not sure about this, i understand that allow me to send the code snippet without declaring a function but if you guys now a cleaner way

Reply:
yes so no it is good since it is a Script command (spawn) that is launched in unscheduled and not a scripted function

Reply:
@Sparker95 if you agree can you resolve ? (and merge or ill merge whatever)